cmd/s3-gw/app.go

```diff
@@ -291,5 +293,15 @@ func getHandlerOptions(v *viper.Viper, l *zap.Logger) *handler.Config {
 			zap.Error(err))
 	}
 
+	if v.IsSet(cfgDefaultMaxAge) {
+		defaultMaxAge = v.GetString(cfgDefaultMaxAge)
+	}
+
+	if _, err = strconv.Atoi(defaultMaxAge); err != nil {
+		l.Fatal("couldn't parse default max age", zap.Error(err))
+	}
+
+	cfg.DefaultMaxAge = defaultMaxAge
```

Comment:
We can also change type of `cfg.DefaultMaxAge` to int

Reply:
I'm not sure we need in keep it as int. Yep, it'd be looking clearer but the the only usage of the variable is to be set in a header as a string. So I decided to use it as a string to avoid some extra itoa.

Reply:
> So I decided to use it as a string to avoid some extra itoa.

If cors are stored we will do itoa anyway.